haskell/defs.bzl

```diff
@@ -91,6 +91,12 @@ _haskell_common_attrs = {
         cfg = "host",
         default = Label("@rules_haskell//haskell:ghc_wrapper"),
     ),
+    "worker": attr.label(
+        default = None,
```

Comment:
It's sad that Bazel doesn't allow `select` as a `default`.

Reply:
I guess you're referring to [this bit](https://github.com/digital-asset/daml/blob/bd686aa6a0151e5cb4074ddf3ac24ceee13e7227/bazel_tools/haskell.bzl#L246). I think it's not so much a workaround for the fact that `select` is not allowed as a `default`. It's more generally a macro wrapper around the `haskell_repl` rule to define some custom defaults in that repository. Since we're not patching the rule definition directly, we don't have access to the the attribute's default value. 

That `select` is not allowed as an attribute's default value, but is allowed in a macro wrapping a rule seems like an arbitrary restriction, indeed. Turns out there's an open issue about it: https://github.com/bazelbuild/bazel/issues/287